### PR TITLE
DPTS-44 Tag and Production pipeline

### PR DESCRIPTION
**Pull request by dehe1**

### CODE REVIEW

File: build/jenkins/Jenkinsfile.production [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    }
+    stage('Deployment Summary') {
+      when { expression { env.PROFILE == 'demo' || env.PROFILE == 'live' } }
+      steps {
+        script { sh 'make deployment-summary' }
+      }
+    }
+  }
+  post {
+    success { sh "make pipeline-send-notification PIPELINE_NAME='DoS Capacity API (Production)' BUILD_STATUS=${currentBuild.currentResult}" }
```
Comment (stefaniuk) [inline build/jenkins/Jenkinsfile.production:40]: Please, can we change it to 'DoS Capacity Status API (Production)'?

File: build/jenkins/Jenkinsfile.production [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    failure { sh "make pipeline-send-notification PIPELINE_NAME='DoS Capacity API (Production)' BUILD_STATUS=${currentBuild.currentResult}" }
```
Comment (stefaniuk) [inline build/jenkins/Jenkinsfile.production:41]: Please, can we change it to 'DoS Capacity Status API (Production)'?

**Review by stefaniuk — CHANGES_REQUESTED** [review]
Just one small thing, Dec, if you can please. Otherwise good to go 👍

File: build/jenkins/Jenkinsfile.production (hunk 2 above)
Comment (dehe1) [reply build/jenkins/Jenkinsfile.production]: Done

**Review by stefaniuk — APPROVED** [review]
Thank you!

**Pull request merged**